.devcontainer/devcontainer.json

```diff
@@ -13,6 +13,7 @@
   "workspaceFolder": "/workspace",
   "remoteEnv": {
     "DATABASE_URL": "postgresql://admin:password@database:5432/dev",
+    "DATABASE_URL_TEST_POSTGRESQL": "postgresql://admin:password@database:5432/dev",
```

Comment:
@mabiede Why are you not just using `DATABASE_URL`?

Reply:
@jerben IMO the test database should not be the same as the development one. So I decided to use the same syntax as in sihl.

Reply:
@mabiede Those env vars were meant to be specific to running the tests of `sihl`. I assume you have a workflow where the dev and test databases are different? Then you can always create your own alias `DATABASE_URL=postgresql://admin:password@database:5432/dev make test` no?

Reply:
@jerben yes, we do. Anyway, I removed the additional variable and check only for the `DATABASE_URL` also when running the tests.